tests/widget/modal/widget-modal.webauthn-login.test.js

```diff
@@ -0,0 +1,40 @@
+import { expect, test } from '@playwright/test';
+import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
+
+test('Modal widget with webauthn login', async ({ browser, page }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
+        hasResidentKey: true,
+      },
+    });
+
+    await navigate('widget/modal?journey=TEST_WebAuthn-Registration');
+
+    await clickButton('Open Login Modal', '/authenticate');
+
+    await page.getByLabel('Username').click();
+    await page.getByLabel('Username').fill('demouser');
+    await page.getByLabel('Username').press('Tab');
```

Comment:
Same thing here, let's just remove the both the click and tab as it's not doing or testing anything.

Reply:
done